### PR TITLE
Email Template and Changes

### PR DESCRIPTION
**Pull request by mailamiton**

### CODE REVIEW

File: handlers/mailHandler.js [hunk 1]
```diff
@@ -14,14 +14,15 @@ var orderTemplateSender = transporter.templateSender(
...
+exports.placeOrder = function (email, username, name, orderDetails) {
     orderTemplateSender({
         to: email,
-        subject: 'Order Alert: You have Purchase Order from : ' + store.get('user').name,
-        html: getHtml(tableBody,username)
-    }, {
-        name: name,
-        username: username
+        subject: 'Order Alert: You have Purchase Order from : ' + store.get('user').name
+        //html: getHtml(tableBody,username)
```
Comment (kumarrakesh) [inline handlers/mailHandler.js]: commented out lines, forgot to delete or may be needed later?

File: public/js/productlist/productlist.js [hunk 2]
```diff
@@ -37,15 +37,31 @@ function createTableElement(parentElem, elem) {
...
+       } else if(elem.name == "GRÖNSAKER") {
+        String += "<tr class='treegrid-"+ elem.id + "'style=background-color:green;'"+">";
+        } else {
+           String += "<tr class='treegrid-"+ elem.id + "'>";
+        }
     } else {
-        String += "<tr class='treegrid-"+ elem.id + " treegrid-parent-"+parentElem + "'>";
+        if(elem.images !== "" && elem.images !== undefined){
+            String += "<tr class='treegrid-"+ elem.id + " treegrid-parent-"+parentElem + "'>";
+            //String += "<tr class='treegrid-"+ elem.id + " treegrid-parent-"+parentElem + "' <img src='"+ elem.images + "' style=width:20px;height:20px;  " + '"</img>"' + '"/>"';
```
Comment (kumarrakesh) [inline public/js/productlist/productlist.js]: commented out lines !

File: views/product-tree.ejs [hunk 3]
```diff
@@ -3,7 +3,8 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="chrome=1">
     <title>Order Management System</title>
-    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
+    <!--<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>-->
```
Comment (kumarrakesh) [inline views/product-tree.ejs]: commented out line!

File: views/product-tree.ejs [hunk 4]
```diff
@@ -16,28 +17,29 @@
   </head>
   <body>
 <div align="center" id="status">
-            <strong>Welcome <%=username%>, Place your order here !</strong>
+            <strong><a style="color:black;font-size:30px;">Welcome <%=username%>, Place your order here !</a></strong>
```
Comment (kumarrakesh) [inline views/product-tree.ejs]: is it possible to move inline css for product-tree.ejs to a separate file ?

File: views/product-tree.ejs [hunk 5]
```diff
@@ -16,28 +17,29 @@
...
         </div>
-    <form method='post' action='/send' onsubmit="submitOrder()">
+  <!-- <form method='post' action='/send' onsubmit="submitOrder()">-->
```
Comment (kumarrakesh) [inline views/product-tree.ejs]: commented out line !

File: views/product-tree.ejs [hunk 6]
```diff
@@ -16,28 +17,29 @@
...
             <input type="hidden" value="" name="orderJson" id="orderJson"/>
-            <input  class="btn btn-primary" type='submit' value='Submit Order'/>
+            <input id = "submitOrderButton" class="btn btn-primary" type='submit' value='Submit Order'/>
         </div>
         <div  align="center" id="status" style="height:20px"></div>
         <div class="footer-copyright text-center py-3" style="padding:20px">© 2018 Copyright:
             <a> EuroCart</a>
         </div>
-    </form>
+    <!--</form>-->
```
Comment (kumarrakesh) [inline views/product-tree.ejs]: commented out line!

File: views/product-tree.ejs [hunk 7]
```diff
@@ -46,15 +48,59 @@
...
-        }
+        
         traverse(jsonObj,"", tableElem);
         document.getElementById("tableElemDisplay").innerHTML=tableElem;
         $('.tree').treegrid({
             'initialState': 'expanded',
             'saveState': true,});
+
+            /********Submit Order *****************/
+            //identify all non empty text. -  Done
```
Comment (kumarrakesh) [inline views/product-tree.ejs]: use jquery filter function to find the non-ordered rows 
and this is a backend processing as compared to in the fronend

File: public/js/productlist/productlist.js (hunk 2 above)
Comment (mailamiton) [reply public/js/productlist/productlist.js]: to be done in Arcticle no + color change + font changes - pull rquest

File: handlers/mailHandler.js (hunk 1 above)
Comment (mailamiton) [reply handlers/mailHandler.js]: Ok .. Deleted..

**Pull request merged**